sdk-manifests/ably-rust.yaml

```diff
@@ -0,0 +1,59 @@
+%YAML 1.2
+---
+common-version: 1.2.0-alpha.1
+compliance:
+  .caveats: |
```

Comment:
This is inventive use of this key at this level, though the build script will be ignoring it. I'm happy for you to leave it here, or you could simply convert it to a YAML comment. I'll leave that up to you. 😁

Reply:
I'd like it to be user visible so I'll keep it like this rather than have it as a comment (even if it's not visible yet)